pyronearEngine/raspberryPi/pyronearEngine.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Pyronear contributors.
+# This file is dual licensed under the terms of the CeCILL-2.1 and AGPLv3 licenses.
+# See the LICENSE file in the root of this repository for complete details.
+
+from picamera import PiCamera
+from time import sleep
+import glob
+import torchvision
+import torch
+from torch import nn
+from PIL import Image
+from torchvision import transforms
+import smtplib
+import ssl
+
+
+class PyronearEngine:
+    """This class is the Pyronear Engine. This engine manage the whole Fire Detection
+       process by capturing and saving the image and by predicting if there is a fire or
+       not based on this image.
+    Example
+    -------
+    pyronearEngine = PyronearEngine()
+    pyronearEngine.run(30)  # For a prediction every 30s
+    """
+    def __init__(self, imgsFolder):
+        # Camera
+        self.camera = PiCamera()
+        self.camera.rotation = 270
+
+        # Images Folder
+        self.imgsFolder = imgsFolder
+
+        # Model definition
+        self.model = torchvision.models.resnet18(pretrained=False)
+
+        # Change fc
+        in_features = getattr(self.model, 'fc').in_features
+        setattr(self.model, 'fc', nn.Linear(in_features, 2))
+
+        self.model.load_state_dict(torch.load("model/pyronear.pth", map_location=torch.device('cpu')))
```

Comment:
I think it would be better to pass the checkpoint path to the constructor, so that the user can easily specify her/his own!